src/Features/ERDDAP/Platform/Observations/Table/all.tsx

```diff
@@ -4,11 +4,17 @@
 import * as React from "react"
 import { ListGroup, ListGroupItem } from "reactstrap"
 
+import { UnitSystem } from "Features/Units/types"
+
 import { RenderProps } from "../../Grabber"
 import { itemStyle, TableItem } from "./item"
 
-export const ErddapAllObservationsTable: React.SFC<RenderProps> = ({ platform, unit_system }: RenderProps) => {
-  const times = platform.properties.readings.filter(d => d.time !== null).map(d => new Date(d.time as string))
+interface Props extends RenderProps {
+  unit_system: UnitSystem
+}
+
+export const ErddapAllObservationsTable: React.SFC<Props> = ({ platform, unit_system }) => {
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)